web_external/InfoView/style.styl

```diff
@@ -0,0 +1,32 @@
+.v-infoview
+  height 100%
+
+  .panel
+    height 100%
+    display flex
+    flex-direction column
+
+  .panel-body
+    flex 1
+    position relative
+
+  ul
+    &.geometry
+      padding-left 20px
+
+    &.activity
+      padding-left 15x
+
+    li
+      &.track
+        margin-bottom 10px
+
+      &.activity
+        margin-bottom 2px
+
+  .clear-message
```

Comment:
Wouldn't it be better to let flex take care of positioning this?

Also, I think I'd like to see `opacity 0.6` added...